goodwe/model.py

```diff
@@ -14,11 +14,17 @@
                        "EHU", "EHR", "HSB",  # ET
                        "ESN", "EMN", "ERN", "EBN", "HLB", "HMB", "HBB", "SPN"]  # ES Gen 2
 
-MPPT3_MODELS = ["MSU", "MST", "PSC", "MSC"]
-
-MPPT4_MODELS = ["HSB",
+MPPT3_MODELS = ["MSU", "MST", "PSC", "MSC",
                 "25KET", "29KET", "29K9ET", "30KET"]
```

Comment:
Would be great if you could provide raw response samples so we can add them to unit tests.
We do not have 29K ET samples yet and these 29KET and 30KET are most likely just wrong, I added them when first found ET >15K models and did not know how goodwe encodes it in serialnr. 29K9ET is the correct one.